test/blackbox-tests/test-cases/env-cflags/run.t

```diff
@@ -0,0 +1,18 @@
+  $ export STANDARD_C_FLAGS=`ocamlc -config | grep ocamlc_cflags | sed 's/ocamlc_cflags: //'`
+  $ export STANDARD_CXX_FLAGS=`ocamlc -config | grep ocamlc_cflags | sed 's/ocamlc_cflags: //' | sed 's/-std=[^ ]* //' | sed 's/-std=[^$]*//'`
```

Comment:
I think CI shows that this doesn't work. Dune starts a new sh process for every `$` so the environment variables aren't saved. I'm not sure how to workaround this except for writing your own .sh files that will wrap the command with these substitutions.